### PR TITLE
main features established on html

### PR DESCRIPTION
**Pull request by PReichetanz**

### CODE REVIEW

File: neuefische.html [hunk 1]
```diff
@@ -9,7 +9,129 @@
...
+                        type="checkbox"
+                        name="newsletter">
+                    <label for="newsletter">Ich willige ein, dass ich per E-Mail Informationen zu Kursen,
+                        Startterminen oder <span>Aktionen</span> von neuefische erhalten werde. Ich kann 
+                        diese Benachrichtigungen jederzeit abbestellen.
+                    </label>
+                    <p>Mit dem Absenden des Formulars erkläre ich mich mit der <a href="datenschutzerklärung.html">Datenschutzerklärung</a>
+                    von neuefische einverstanden.</p>
+                </div>
+                <button>Jetzt Informationen anfordern</button>
```
Comment (derjubu) [inline neuefische.html:132]: Better use input type="submit"

File: neuefische.html [hunk 2]
```diff
@@ -9,7 +9,129 @@
...
+                    placeholder="z. B. Müller, Meier oder Schmidt" 
+                    required><br>
+                <label for="email">E-Mail-Adresse*</label>
+                <input 
+                    id="email" 
+                    type="email" 
+                    placeholder="z. B. honey2000@gmx-net.de" 
+                    required>
+                
+
```
Comment (derjubu) [inline neuefische.html:50]: This line could be deleted

**Comment by PReichetanz** [comment]
Chanced button to <input type="submit">

File: neuefische.html [hunk 3]
```diff
@@ -9,7 +9,129 @@
...
+                    <span>Newsletter</span>
+                    <input 
+                        id="newsletter"
+                        type="checkbox"
+                        name="newsletter">
+                    <label for="newsletter">Ich willige ein, dass ich per E-Mail Informationen zu Kursen,
+                        Startterminen oder <span>Aktionen</span> von neuefische erhalten werde. Ich kann 
+                        diese Benachrichtigungen jederzeit abbestellen.
+                    </label>
+                    <p>Mit dem Absenden des Formulars erkläre ich mich mit der <a href="datenschutzerklärung.html">Datenschutzerklärung</a>
```
Comment (derjubu) [inline neuefische.html:129]: <a href="datenschutzerklaerung.html"> would be better

**Pull request merged**